src/applications/disability-benefits/all-claims/pages/reservesNationalGuardService.jsx

```diff
@@ -12,11 +16,31 @@ export const uiSchema = {
   'ui:description': ReservesGuardDescription,
   serviceInformation: {
     reservesNationalGuardService: {
-      obligationTermOfServiceDateRange: dateRangeUI(
-        'Obligation start date',
-        'Obligation end date',
-        'End date must be after start date',
-      ),
```

Comment:
I don't want to update the shared component, so I'm "un-abstracting" the whole thing here.